coralproject/docs
# Docker-Compose: Add Xenia's conditional environment variables

## What does this PR do?

Modifies the `compose.yml` file to add additional environment variables dependent on some changes in https://github.com/coralproject/xenia/pull/129.

## How do I test this PR?
To see default behavior hasn't changed.
1. `docker-compose up xeniaapp`

The `xeniad` application should start up and begin listening. No other command logs should be produced.

To see new changes 
1. Export XENIA_CREATE_DATABASE or XENIA_UPDATE_QUERY so they're not empty.
2. `docker-compose up xeniaapp`

The `xenia` cli tool should be ran prior to the `xeniad` daemon.
